workflow-trigger

```diff
@@ -2,4 +2,5 @@
 2022-02-25T18:58:00Z
 2022-02-25T21:07:00Z
 2022-04-03T14:29:00Z
-2022-04-06T11:33:00Z
\ No newline at end of file
+2022-04-06T11:33:00Z
+2022-04-06T12:02:00Z
```

Comment:
@123MwanjeMike there is no need to trigger the entire workflow since the `network-uptime` folder has changes that will trigger its workflow

Reply:
Thanks @NoahNsimbe. Have undone the change